src/patterns/download-resources/index.njk

```diff
@@ -0,0 +1,13 @@
+---
+title: Download resources
+group: Help users to...
+experimental: true
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+
+Allow users to filter and sort downloadable resources.
+
+## Example
+{{
+    patternlibExample({ "path": "patterns/download-resources/examples/download-resources/index.njk" })
+}}
```

Comment:
```suggestion
}}

## Research on this component
{% call onsPanel() %}
    If you have conducted any user research using this component, please feed back your findings via the
    {{
        onsExternalLink({
            "url": "https://github.com/ONSdigital/design-system/issues/45",
            "linkText": "Design System forum"
        })
    }}
{% endcall %}

## Design System forum
{{
    onsExternalLink({
        "url": "https://github.com/ONSdigital/design-system/issues/45",
        "linkText": "Discuss 'Download Resources' on GitHub"
    })
}}
```

Youll have to create an issue and change the number on the links in this suggestion

Reply:
Alright will take a look at this.

Reply:
This has been added.